syntax="proto3";
package deployer;

message Exec {
    string cwd = 1;
    string cmd = 2;
}

message Stdout {
    string status = 1;
    string result = 2;
}

message DeployerRequest{
    repeated Exec shellExec = 1;
}

message DeployerResponse{
    Stdout stdout = 1;
}

service DeployerTools{
    rpc ExecCMD (DeployerRequest) returns (stream DeployerResponse){};
}
